### DISCUSSION TITLE
Satellite addon bug

### DISCUSSION THREAD

**Discussion by jairoamaral** [Stellarium/stellarium#3507] [Q&A] [answered]
A few months ago, the satellite addon began to display some unrealistically fast sattelites, ocasionally zapping throught the screen. It was never fixed in the updates.

Maybe the database is corrupted. I wouldn't like to reinstall Stellarium from zero, so how do I make a fresh install of this addon?

**Comment by gzotti** [comment, marked as answer]
It should be enough to reset/update the satellite data.

**Comment by jairoamaral** [comment]
Updating the satellite data didn't work, but using the option to return to original configurations (going back to data from years ago) solved it. Then I just updated the data again. Thank you!
